timemachine/fe/free_energy.py

```diff
@@ -402,10 +402,17 @@ def sample_with_context(
 ) -> Trajectory:
     # burn-in
     if md_params.n_eq_steps:
+        barostat = ctxt.get_barostat()
+        default_interval = 0 if barostat is None else barostat.get_interval()
+        equil_barostat_interval = 15
```

Comment:
Do we think it's really necessary to use different intervals for equilibration and production? Seems simpler to just switch to 25 in both places unless we have evidence that this is insufficient for equilibration.

Reply:
nit, if we decide to keep the shorter interval for equilibration: I think a short comment would be useful here, e.g. "set to 15 steps per barostat move for equilibration, then back to the original interval for production"

Reply:
Good question, I would defer to @proteneer on this. 

My intuition is that we want to be 'sure' that the system is at equilibrium before collecting samples and setting a lower interval before collecting samples helps us be certain.

Reply:
Added comment https://github.com/proteneer/timemachine/pull/1221/commits/abe24775b89b2983c16d0af4f4fa10bb0ba84695, until @proteneer comments on the utility of having two different intervals.

Reply:
I think the idea was to initially have a higher rate of equilibration when the system may be out  equilibrium (eg. we pad the water box so the density is too low). I don't have a quantitative sense of 15 vs 25 though, I think we'd need to monitor volumes/densities to be more precise here.